docs/beast_development.rst

```diff
@@ -211,8 +207,53 @@ The resulting HTML files are placed in `docs/_build/html` subdirectory, and
 can be viewed in a web browser.
 
 
-Collaborating and Contributing
-==============================
+Submitting a Pull Request
+=========================
+
+Once you have changes that you'd like to contribute back to the project or share
+with collaborators, you can open a pull request. It is a good idea to check with
+the projects or your collaborators which branch of their BEAST repository you
+should send the pull requests.

```

Comment:
Isn't it always the BEAST-Fitting/beast master branch?

Reply:
True, I just just re-added this text. Will clarify.